Messaging.md

```diff
@@ -87,33 +97,123 @@ The following DLC-specific types are used throughout the specification. All type
 
 ### The `contract_info` Type
 
-This type contains information about a contracts outcomes and their corresponding payouts. To save space, only one side's POV is included in this message as the other can be derived using `remote_payout = total_collateral - local_payout`.
+This type contains information about a contract's outcomes, their corresponding payouts, and the oracles to be used.
 
 #### Version 0 `contract_info`
 
-1. type: 42768 (`contract_info_v0`)
+1. type: 55342 (`contract_info_v0`)
```

Comment:
Just to make sure, we are keeping v0 but changing it is that ok? I guess it should be fine since we don't need to be backward compatible at that point but just wanted to make sure.

Reply:
Yes, the old type is now a `contract_descriptor_v0` which is what the old contract info was at a byte level

Reply:
understood